adds language support for Wikipedia results and revert implementation of docker logs using 'docker logs -f librex'

juniorbotelho opened the pull request:
This PR adds support for other languages in Wikipedia librex results, as reported in issue #168. Also, now the user can keep track of error and access logs of `nginx/php` using the `docker logs -f librex` command.

- Adds language support for Wikipedia Librex Results (See: #168)
- Add support for logs using `docker logs -f librex` command

**Attention:** If the user omits the Wikipedia language environment variable, it will use the language set by the Google search engine

New commands for Docker Wiki from LibreX:

```sh
docker run -d \
  --name librex \
  -e TZ="America/New_York" \
  -e CONFIG_GOOGLE_DOMAIN="com" \
  -e CONFIG_GOOGLE_LANGUAGE="en" \
  -e CONFIG_WIKIPEDIA_LANGUAGE="en" \
  -p 8080:8080 \
  librex/librex:latest
```

```
version: "2.1"
services:
  librex:
    image: librex/librex:latest
    container_name: librex
    network_mode: bridge
    ports:
      - 8080:8080
    environment:
      - PUID=1000
      - PGID=1000
      - VERSION=docker
      - TZ="America/New_York"
      - CONFIG_GOOGLE_DOMAIN="com"
      - CONFIG_GOOGLE_LANGUAGE="en"
      - CONFIG_WIKIPEDIA_LANGUAGE="en"
    volumes:
      - ./nginx_logs:/var/log/nginx
      - ./php_logs:/var/log/php7
    restart: unless-stopped
```

juniorbotelho commented:
I forgot something, sorry!

hnhx commented:
> "wikipedia_language": "en",

Should be.

> "wikipedia_language" => "en",

To keep the codebase consitent.

hnhx commented:
Also I'm not sure if the logging option is a good idea, LibreX should be as private as possible.
I know that by default nginx logs things, but still, turning it on on purpose is kinda against this whole project.

juniorbotelho commented:
You are correct. I removed that implementation and fixed the consistency of the codebase. As of now, only the 'Wikipedia language' feature is included in this PR.

hnhx commented:
Thanks
